### ISSUE TITLE
query results cannot span epoch 0

### ISSUE THREAD

**Issue by beckettsean** [influxdata/influxdb#2703]
Here are my points:

```
{"database":"mydb",
"retentionPolicy":"default",
"points":[
    {"name":"old",
	 "time": -1422,
	 "precision": "s",
         "fields":{"value":10000000}},
    {"name":"old",
	 "time": 1422568543,
	 "precision": "ms",
	 "fields":{"value":10000}},
    {"name":"old",
	 "time": 1422568543,
	 "precision": "s",
         "fields":{"value":100}},
    {"name":"old",
	 "fields":{"value":1}}]
}
```

One point is written to a time before epoch 0, one is written a few weeks after epoch 0, one is earlier this year, and the final one inherits now() when inserted.

I cannot craft a `WHERE time` clause that will return all points:

```
> select * from old
name: old
---------
time				value
1970-01-17T11:09:28.543Z	10000
2015-01-29T21:55:43Z		100
2015-05-29T23:23:01.796811813Z	1

> select * from old where time > -10000s
name: old
---------
time			value
1969-12-31T23:36:18Z	1e+07

> select * from old where time < now()
name: old
---------
time				value
1970-01-17T11:09:28.543Z	10000
2015-01-29T21:55:43Z		100
2015-05-29T23:23:01.796811813Z	1

> select * from old where time > -10000s and time < now()
name: old
---------
time			value
1969-12-31T23:36:18Z	1e+07
```

**Comment by beckettsean** [comment]
Same appears to hold for aggregate queries, as well:

```
> select count(value) from old 
name: old
---------
time			count
1970-01-01T00:00:00Z	3

> select count(value) from old where time > -10000s 
name: old
---------
time			count
1970-01-01T00:00:00Z	1

> select count(value) from old where time < now()
name: old
---------
time			count
1970-01-01T00:00:00Z	3

> select count(value) from old where time > -10000s and time < now()
name: old
---------
time			count
1970-01-01T00:00:00Z	1
```

**Comment by jwilder** [comment]
Fixed via #7212.
